src/views/notifications/Create.vue

```diff
@@ -0,0 +1,323 @@
+                        <div class="w-20 h-20 sm:w-24 sm:h-24 flex-none lg:w-32 lg:h-32 image-fit relative">
+                          <img
+                            alt=""
+                            class="rounded-full"
+                            :src="this.create_notification.user?.profile_picture ?? require('@/assets/images/avatar.png')"
+                          />
+                        </div>
+                        <div class="ml-5">
+                          <div class="w-24 sm:w-40 truncate sm:whitespace-normal font-medium text-lg">
+                            {{ this.create_notification.user?.name }}
+                          </div>
+                          <div class="text-gray-600">{{ this?.create_notification?.user.role?.name }}</div>
+                        </div>
+                      </div>
+                      <div class="mt-6 lg:mt-0 flex-1 dark:text-gray-300 px-5 border-l border-r border-gray-200 dark:border-dark-5 border-t lg:border-t-0 pt-5 lg:pt-0">
+                        <div class="font-medium text-center lg:text-left lg:mt-3">
+                          Contact Details
+                        </div>
+                        <div class="flex flex-col justify-center items-center lg:items-start mt-4">
+                          <div class="truncate sm:whitespace-normal flex items-center">
+                            <MailIcon class="w-4 h-4 mr-2"/>{{ this?.create_notification?.user.email }}
+                          </div>
+                          <div class="truncate sm:whitespace-normal flex items-center mt-3">
+                            <UserIcon class="w-4 h-4 mr-2"/>{{ this?.create_notification?.user?.pre_name }} {{ this?.create_notification?.user?.last_name }}
+                          </div>
+                          <div class="truncate sm:whitespace-normal flex items-center mt-3">
+                            <HashIcon class="w-4 h-4 mr-2"/>User ID: {{ this?.create_notification?.user.id }}
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div class="p-5">
+                  <div class="flex flex-col-reverse xl:flex-row flex-col">
+                    <div class="flex-1 mt-6 xl:mt-0">
+                      <div class="grid grid-cols-12 gap-x-5 mb-3">
+                        <div class="col-span-12 xxl:col-span-12 mb-4">
+                          <div>
+                            <label for="create-title-notification" class="form-label">
+                              Title
+                            </label>
+                            <input
+                              id="create-title-notification"
+                              type="text"
+                              :class="'form-control' + (this.validation_error?.title != null ? ' border-theme-6' : '')"
```

Comment:
Unnecessary `!= null`

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/34839e538cf6eea37658b6eebcf618108f1b8fc8